packages/sanity/src/desk/panes/document/timeline/__workshop__/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+      "i18next/no-literal-string": "off"
+    }
+}
```

Comment:
Nitpick, but preferably add newlines at the end of files (the [EditorConfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) VS Code extension automatically does this for you based on our config, fwiw).